docs/modules/operation/pages/quick-start/performance-data.adoc

```diff
@@ -0,0 +1,50 @@
+
+= Step 4: Set Up Performance Data Collection
+
+This section describes how to set up performance data collection in {page-component-title}.
+
+{page-component-title} collects performance data using xref:reference:daemons/daemon-config-files/collectd.adoc[collectd], which is enabled by default.
+Collectd schedules data collection on network entities using management agents and protocol-specific collectors.
+
+NOTE: In the {page-component-title} docs, any reference to collectd refers to the {page-component-title} collect daemon, not the open source project of the same name.
+
+Data collection for SNMP and OpenNMS-JVM is enabled by default, allowing {page-component-title} to monitor itself through JMX.
+
+For additional information, including collector configuration, see xref:operation:deep-dive/performance-data-collection/introduction.adoc[Performance Management] in the Deep Dive section.
+
+== Before you begin
+
+{page-component-title} data collection for SNMP works out of the box, provided your SNMP credentials are configured correctly.
+
+The default value for the community string is `public`.
+
+NOTE: If you are using SNMPv3, you need to configure details for a User-Based Security Model (USM) user instead of a community string.
+{page-component-title} supports SNMPv3 and USM, but these topics are outside of the scope of this guide.
+
+If your community string is different, you need to change it by following these steps:
+
+. In the web UI, click the *Gear* symbol at the top-right of the screen.
+. Choose menu:Provisioning[Configure SNMP Community Names by IP Address].
```

Comment:
Insert missing steps (I just caught this, sorry it's last-minute):
- Enter a valid IP address and click *Look up*.
- In the *Updating SNMP Configuration* section, choose `v2c` or `v1` for *Version* as appropriate for your environment
- If you wish to configure SNMP credentials for a range of addresses, set the *First IP Address* and *Last IP Address* inputs to describe the desired range

Reply:
```suggestion
. Choose menu:Provisioning[Configure SNMP Community Names by IP Address].
. Set up the configuration as appropriate for your environment:
** Enter a valid IP address in the *IP Address* box, and click *Look Up*.
** Choose an SNMP version (`v2c` or `v1`) from the *Version* list.
** (Optional) Define a range of IP addresses by typing the first and last addresses of the range into the *First IP Address* and *Last IP Address* boxes, respectively.
```